### ISSUE TITLE
HomeGuest Component Not Displaying Full-Width Container

### ISSUE THREAD

**Issue by Aishwary-Pratap-Singh** [Aishwary-Pratap-Singh/complex-react-app#4]
The HomeGuest component should render a full-width container, but it is incorrectly displaying a narrow container due to the wide prop not being passed correctly from the Page component to the Container component.

**Steps to Reproduce:**

Import and render the HomeGuest component in your application.
Observe the width of the container

**Expected Behavior:** The HomeGuest component should render with a full-width container (without the container--narrow class).

**Actual Behavior:** The HomeGuest component renders with a narrow container (with the container--narrow class).

![homepage-complex app](https://github.com/Aishwary-Pratap-Singh/complex-react-app/assets/66994420/31c570f3-6274-4b7e-a284-c40965845467)

**Comment by Aishwary-Pratap-Singh** [comment]
**Cause:** 
The wide prop is not being passed from the Page component to the Container component.

**Solution:**
Modify the Page component to pass the wide prop to the Container component.

**Comment by Aishwary-Pratap-Singh** [comment]
**Resolution Summary:**
The issue with the HomeGuest component displaying a narrow container has been resolved. The fix involved ensuring that the wide prop is correctly passed to the Container component from the Page component.

**Pull Request:**
Link to the PR that fixed the issue: #5

**Additional Context:**
We conducted thorough testing to validate that the component now renders correctly across different screen sizes and conditions.

Thank you to everyone who contributed and reviewed the changes!

![homepage-complex app](https://github.com/Aishwary-Pratap-Singh/complex-react-app/assets/66994420/97cb6fe7-3103-4e54-8aab-6be4105adf17)

Closing this issue now.
